### DISCUSSION TITLE
If you have multiple webhooks (like with a Zapier integration) you can't tell which webhook are working and which ones aren't

### DISCUSSION THREAD

**Discussion by woutersamaey** [btcpayserver/btcpayserver#2616] [Technical Support] [answered]
I have a store with over 10 webhooks.
An invoice is created and a whole lot of webhooks start firing.

1 Webhook succeeds. The others don't and are retied a number of times.

I can't tell which webhook is the good one and which one's aren't working. This makes cleanup & fixing impossible without removing all webhooks and starting fresh :( 

Proposed solution: Can we add an icon to the webhook that failed last time? Like a red warning sign?
Or even better: show a counter of succesfull deliveries and failed deliveries in the last hour / day / week ?

Invoice looks like this:
![image](https://user-images.githubusercontent.com/71019/123278076-f01ec300-d506-11eb-89b6-80eaa8f2ab65.png)

Webhooks look like this (though we have an improved UI coming that shows longer URLs... but still this does not help me tackle this):
![image](https://user-images.githubusercontent.com/71019/123278213-0c226480-d507-11eb-9aba-4e4e1b2d634c.png)

**Comment by dstrukt** [comment]
Concept ACK.

Not sure the technical work involved in the latter (counter), but the icon is definitely helpful.

**Comment by bolatovumar** [comment]
This should be doable. I can take a look into it.

**Reply by woutersamaey** [reply]
Yes please. Shout if you need any help.

**Reply by bolatovumar** [reply, marked as answer]
PR here: https://github.com/btcpayserver/btcpayserver/pull/2679
